polus-tiledtiff-converter-plugin/README.md

```diff
@@ -26,20 +38,17 @@ To build the Docker image for the conversion plugin, run
 
 ## Install WIPP Plugin
 
-If WIPP is running, navigate to the plugins page and add a new plugin. Paste the contents of `plugin.json` into the pop-up window and submit.
+If WIPP is running, navigate to the plugins page and add a new plugin. Paste the
+contents of `plugin.json` into the pop-up window and submit.
 
 ## Options
 
 This plugin takes one input argument and one output argument:
 
 | Name       | Description             | I/O    | Type |
 |------------|-------------------------|--------|------|
-| `input`   | Input image collection  | Input  | Path |
-| `tileSizeX`| Tile width             | Input  | Number |
-| `tileSizeY`| Tile height            | Input  | Number |
-| `output`   | Output image collection | Output | List |
-
-_Note:_ All tile sizes must be a positive integer than is a multiple of 16
+| `input`    | Input image collection  | Input  | Path |
+| `output`   | Output image collection | Output | Path |
```

Comment:
plugin.json states that the plugin has another input called `useMetadata` which is missing here.